tox.ini

```diff
@@ -68,6 +68,7 @@ deps =
     cosl
     fs
     pytest
+    pytest-asyncio
```

Comment:
let's add a comment about why we need this

Reply:
I don't remember which warning it was clearing :)